ReactNativeClient/lib/services/CommandService.ts

```diff
@@ -1,4 +1,5 @@
 const BaseService = require('lib/services/BaseService');
+const KeymapService = require('lib/services/KeymapService.js');
```

Comment:
Since KeymapService is a TypeScript file, you should import it with `import KeymapService from "..."` so as to benefit from type-checking.